src/ts/components/playbacktogglebutton.ts

```diff
@@ -39,11 +39,17 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
 
       if (player.isPlaying() || this.isPlayInitiated) {
         this.on();
+        updateAriaLabel('pause');
       } else {
         this.off();
+        updateAriaLabel('play');
       }
     };
 
+    const updateAriaLabel = (label: string) => {
+      this.getDomElement().attr('aria-label', i18n.performLocalization(i18n.getLocalizer(label)));
+    };
```

Comment:
Every component in our UI should in theory support the `ariaLabel`. Therefore this 'update' possibility should be moved to the `component.ts`.

Also, I think the `updateAriaLabel` function shouldn't hide the dependency to the localization feature. I would prefer that this function takes a `LocalizableText` as an input parameter.

Reply:
@stonko1994 I had a stab at this here: https://github.com/bitmovin/bitmovin-player-ui/compare/feature/fix-pause-aria-label...feature/fix-pause-aria-label-2

Reply:
I had a look at it but I'm not super happy with the changes to the `ToggleButtonConfig` 🤔 

I can see a problem that the `ToggleButtonConfig` extends from the `ButtonConfig` which again extends the `ComponentConfig`. That means that the `ToggleButtonConfig` now has 3 `areaLabel` related properties:

```ts
interface ToggleButtonConfig extends ButtonConfig {
  ariaLabel?: LocalizableText; // From the `ComponentConfig`
  onAriaLabel?: LocalizableText;
  offAriaLabel?: LocalizableText;
}
```

IMO that is now super confusing what to use when someone creates a custom component or initializes a `ToggleButton` with a custom configuration 🤔 

_+1: The mentioned bug from the review comment is still there_

Reply:
Fully agree, I wasn't sure what to do though, so I just left it to see what you'd say.

I can see two options:

```
interface ToggleButtonConfig extends ButtonConfig {
  ariaLabel?: LocalizableText; // From the `ComponentConfig`
  offAriaLabel?: LocalizableText;
}
```
and doing some funky type magic to replace the ariaLabel in the inherited class with something like

```
ariaLabel: { on?: LocalizableText, off?: LocalizableText }
```

Reply:
Hm I'm not sure either 🤔 Both not ideal.

Do we really need the configuration option? What about manually managing it for the `ToggleButton` 🤔

Reply:
Discussion happened on Slack. All points resolved in #431